File: baekjoon/boj_2525.cpp
#include <iostream>
using namespace std;

int main()
{
    int hour, minute, extra;

    cin >> hour >> minute >> extra;

    minute += extra;
    while (minute >= 60)
    {
        minute -= 60;
        hour += 1;
    }
    while (hour >= 24)
    {
        hour -= 24;
    }

    cout << hour << ' ' << minute << endl;

    return 0;
}

File: baekjoon_all/02000+/boj_2525.java
// Solve 2023-11-17

// 백준에 제출할 때는 class 이름을 Main으로 설정해야 함.

import java.util.Scanner;

public class boj_2525 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int hour = sc.nextInt();
        int minute = sc.nextInt();
        int extraTime = sc.nextInt();

        minute += extraTime;
        hour += minute / 60;
        minute %= 60;
        hour %= 24;

        System.out.println(hour + " " + minute);

        sc.close();
    }
}
